docs/config.yaml

```diff
@@ -113,19 +115,27 @@ s3:
 # OPTIONAL: container registry-related keys
 registry_repos:
   # OPTIONAL: repo to which to push oscontainer
-  oscontainer: quay.io/fedora/fedora-coreos
+  oscontainer:
+	registry: quay.io/fedora/fedora-coreos
+    tags: ["${STREAM}"]
```

Comment:
I know it seems obvious since we're using it in the examples, but for completeness, can we explicitly document the supported variables?

Reply:
I did in the first instance; didn't for the others.

Reply:
We could even drop the `REQUIRED` comments in the following ones, but cool as is too!